gravity.sh

```diff
@@ -440,7 +440,7 @@ gravity_WhitelistBLD() {
 
   # Create array of unique $sourceDomains
   mapfile -t uniqDomains <<< "$(awk '{ if(!a[$1]++) { print $1 } }' <<< "$(printf '%s\n' "${sourceDomains[@]}")")"
-  [[ "${#uniqDomains[@]}" -ne 1 ]] && plural="s"
+  [[ "${#uniqDomains[@]}" != 1 ]] && plural="s"
 
   str="Adding ${#uniqDomains[@]} blocklist source domain${plural} to the whitelist"
```

Comment:
This line could be
`str="Adding $(printf "%'.0f" "${#uniqDomains[@]}") blocklist source domain${plural} to the whitelist"`